src/types.ts

```diff
@@ -350,7 +350,7 @@ export type RootStackParamList = {
     title?: string;
   };
   transactionAccount: {
-    from: string;
+    to: string;
```

Comment:
Why? This was a typo?

Reply:
<img width="400" alt="image" src="https://github.com/haqq-network/haqq-wallet/assets/25716589/b6d219a9-e66c-4169-8ffe-a8187453a848">

`from` overrides and never use on this screen. This screen have only `to` param.